src/libraries/Native/Unix/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.14.5)
+cmake_minimum_required(VERSION 3.6.3)
```

Comment:
can you please add a check for cmake > 3.14.5 when targetting iOS/tvOS like we had in an earlier version of the file: https://github.com/dotnet/runtime/blob/a56d6a8034f5cb5ad58f9e29eb61edcc792f4b86/src/libraries/Native/Unix/CMakeLists.txt#L8-L11